src/tools/fuzzing/fuzzing.cpp

```diff
@@ -1471,7 +1472,7 @@ Expression* TranslateToFuzzReader::makeGlobalSet(Type type) {
     return makeTrivial(Type::none);
   }
   auto name = pick(it->second);
-  if (isValidGlobal(name)) {
+  if (isValidGlobal(name) && wasm.getGlobal(name)->mutable_) {
```

Comment:
Maybe we should track `mutableGlobalsByType` as well to make it more likely that we successfully create a `GlobalSet` here.

Reply:
Good idea, done.